tests/tools/importexport/migration/test_migration.py

```diff
@@ -13,12 +13,13 @@
 
 from aiida import orm
 from aiida.backends.testbase import AiidaTestCase
-from aiida.backends.tests.utils.archives import get_archive_file, get_json_files, migrate_archive
-from aiida.backends.tests.utils.configuration import with_temp_dir
 from aiida.tools.importexport import import_data, EXPORT_VERSION as newest_version
 from aiida.tools.importexport.migration import migrate_recursively, verify_metadata_version
 from aiida.common.utils import Capturing
 
+from ....utils.archives import get_archive_file, get_json_files, migrate_archive
```

Comment:
that's 4 dots which makes things hard to read... is there a better way to do this?

Reply:
Yeah, as noted in the commit message this is a temporary fix in order to have minimal changes and to make it easy to track where stuff went. Most of these can/should be reimplemented as pytest fixtures which would get rid of the relative imports

Reply:
wouldn't it be possible to `from tests.path.to.module import get_archive`?

The tests should anyhow always be run from the top-level directory of the git repo

Reply:
Done